MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/ReflectiveSerializer.cs

```diff
@@ -127,6 +134,11 @@ protected internal override void Initialize(IntermediateSerializer serializer)
             }
         }
 
+        public override bool CanDeserializeIntoExistingObject
+        {
+            get { return TargetType.IsClass; }
```

Comment:
Hmm, I think I've uncovered an edge-case-bug in XNA - it will happily serialize getter-only properties if the type of the property is `IList`, but will throw an exception when you try to deserialize (that's correct, you shouldn't be able to deserialize into a getter-only array property, whatever the property type may be declared as).

I think we could be a bit cleverer, and not serialize those properties in the first place. That would be a change from XNA behaviour, but this is really an edge case, and I don't think the XNA behaviour is correct. To do that, we'll have to do two checks - once when we're deciding which elements to serialize in `ReflectiveSerializer.Initialize`, and once in `ReflectiveSerialize.Serialize`, when we know the actual type of the object.